Cubical/Codata/M.agda

```diff
@@ -32,27 +32,30 @@ module Helpers where
 
 open Helpers
 
-IxCont : Type₀ → Type₁
-IxCont X = Σ (X → Type₀) \ S → ∀ x → S x → X → Type₀
+IxCont : ∀ {ℓ} -> Type ℓ → Type (ℓ-suc ℓ)
+IxCont {ℓ} X = Σ (X → Type ℓ) λ S → ∀ x → S x → X → Type ℓ
 
 
-⟦_⟧ : ∀ {X : Type₀} → IxCont X → (X → Type₀) → (X → Type₀)
-⟦ (S , P) ⟧ X x = Σ (S x) \ s → ∀ y → P x s y → X y
+⟦_⟧ : ∀ {ℓ} {X : Type ℓ} → IxCont X → (X → Type ℓ) → (X → Type ℓ)
+⟦ (S , P) ⟧ X x = Σ (S x) λ s → ∀ y → P x s y → X y
 
 
-record M {X : Type₀} (C : IxCont X) (x : X) : Type₀ where
+record M {ℓ} {X : Type ℓ} (C : IxCont X) (x : X) : Type ℓ where -- Type₀
```

Comment:
How hard is it to prove an equivalence with your M-type and this one?

Reply:
There are some issues with the termination checker, and how to define functions over the M-types defined in this PR, but it should hopefully not be too hard, I have an attempt at it in my local work.

Reply:
Great! Sounds like a good future PR